dataflow/src/main/java/org/checkerframework/dataflow/analysis/FlowExpressions.java

```diff
@@ -721,17 +721,36 @@ public boolean isUnmodifiableByOtherCode() {
         }
     }
 
+    /** FlowExpression representing a local variable. */
     public static class LocalVariable extends Receiver {
+
+        /** Element of representing the local variable. */
         protected final Element element;
+        /** Hashcode */
+        private final int hashcode;
 
-        public LocalVariable(LocalVariableNode localVar) {
-            super(localVar.getType());
-            this.element = localVar.getElement();
+        /** Create a {@link LocalVariable} from {@code node}. */
+        public LocalVariable(LocalVariableNode node) {
+            super(node.getType());
+            this.element = node.getElement();
+            VarSymbol vs = (VarSymbol) this.element;
+            this.hashcode =
+                    Objects.hash(
+                            vs.name.toString(),
+                            TypeAnnotationUtils.unannotatedType(vs.type).toString(),
+                            vs.owner.toString());
         }
 
-        public LocalVariable(Element elem) {
-            super(ElementUtils.getType(elem));
-            this.element = elem;
+        /** Create a {@link LocalVariable} from {@code element}. */
+        public LocalVariable(Element element) {
+            super(ElementUtils.getType(element));
+            this.element = element;
+            VarSymbol vs = (VarSymbol) this.element;
```

Comment:
Instead of duplicating this logic, could you please introduce a helper method or helper constructor that extracts this?